lib/elixir/test/elixir/fixtures/consolidation/extra_def.ex

```diff
@@ -0,0 +1,5 @@
+defprotocol Protocol.ConsolidationTest.ExtraDef do
+  def protocol_fun(term)
+
+  Kernel.def(regular_fun(term), do: term + 1)
```

Comment:
Or I can add it to `NoImpl` if you prefer to avoid fixture proliferation?

Reply:
Yes, let's add it to to an existing one with a comment on top.

Reply:
Done [9adad46](https://github.com/elixir-lang/elixir/pull/14191/commits/9adad463b95a828d233739707468daf3bcad2ad1)